website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -1075,9 +1075,13 @@ On AL2, Karpenter automatically configures the disks through an additional boost
 
 On AL2023, Karpenter automatically configures the disks via the generated `NodeConfig` object. Like AL2, the device name is `/dev/md/0` and its mount point is `/mnt/k8s-disks/0`. You should ensure any additional disk setup does not interfere with these.
 
+#### Bottlerocket
+
+On Bottlerocket, Karpenter automatically configures the disks by adding the required bootstrap commands to the settings. The ephemeral storage configuration is handled automatically, so you should not manually configure ephemeral storage settings in your userData.
```

Comment:
Can we also add that `this userData configuration is only valid on Bottlerocket v1.22.0+`.